### PR TITLE
Multiple brands throws exception 

### LINKED ISSUE #4: Exception when multiple brand wikidata is used
**Exception**
```
'NoneType' object is not subscriptable
Traceback (most recent call last):
  File "/Users/kasper/dev/OSM-wikipedia-tag-validator/script.py", line 281, in update_problem_for_entry
    reported = issue_detector.get_the_most_important_problem_generic(tags, location, object_type, object_description)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/wikibrain/wikimedia_link_issue_reporter.py", line 233, in get_the_most_important_problem_generic
    something_reportable = self.freely_reorderable_issue_reports(object_description, location, tags)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/wikibrain/wikimedia_link_issue_reporter.py", line 443, in freely_reorderable_issue_reports
    something_reportable = self.check_is_object_brand_is_existing(tags)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/wikibrain/wikimedia_link_issue_reporter.py", line 669, in check_is_object_brand_is_existing
    if self.is_brand_nonexisting(present_wikidata_id):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/wikibrain/wikimedia_link_issue_reporter.py", line 643, in is_brand_nonexisting
    no_longer_existing = wikimedia_connection.get_property_from_wikidata(present_wikidata_id, 'P576')
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/wikimedia_connection/wikimedia_connection.py", line 245, in get_property_from_wikidata
    return wikidata['entities'][wikidata_id]['claims'][property]
           ~~~~~~~~^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
```

After a bit more digging, I found it is caused by https://www.openstreetmap.org/node/9410011527.

I can not find documentation if brand:wikidata can contain multiple wikidata brands (the wiki specifies that you can have multiple brands, which is why I assume it should be possible)  
It is used 100 times in Europe: https://overpass-turbo.eu/s/1FNT

### PR DESCRIPTION
**Pull request by KasperFranz**
This PR showcases the bug #4  and fixes #4

### CODE REVIEW

**Comment by matkoniecz** [comment]
what happens when one of them IS nonexisting? Maybe it would be better to skip this test entirely with multiple brands or split earlier, so report about nonexisting brand will actually report problematic one not merged list of them?

**Comment by KasperFranz** [comment]
> what happens when one of them IS nonexisting? Maybe it would be better to skip this test entirely with multiple brands or split earlier, so report about nonexisting brand will actually report problematic one not merged list of them?

With this PR, it would report the brand as nonexisting if 1 of them are invalid - I am not sure how it would look in the report, I will create a quick test case for it to see output

**Comment by KasperFranz** [comment]
@matkoniecz  do you have an example of P576 with P1011?

I would like to add a test case to ensure it is covered

**Comment by KasperFranz** [comment]
I have changed the underlying function to return the dissolved brands instead, and it is now reporting the error on the invalid wikidata IDs. 

Let me know if you want to adjust the error :)

File: wikibrain/wikimedia_link_issue_reporter.py [hunk 1]
```diff
@@ -691,7 +696,7 @@ def check_is_object_brand_is_existing(self, tags):
                 error_id=state,
                 error_general_intructions=error_general_intructions,
                 error_message=message,
-                prerequisite={'wikidata': present_wikidata_id}
+                prerequisite={'wikidata': ';'.join(dissolved_brands)}
```
Comment (matkoniecz) [inline wikibrain/wikimedia_link_issue_reporter.py:699]: one problem: `prerequisite` is for matching whether such tag is there (maybe I should have documented it?) so QA filter/bot can easily discard cases where data is stale.

Here it should be present_wikidata_id

Maybe info which ones failed can go into `message` field?

Comment (matkoniecz) [reply wikibrain/wikimedia_link_issue_reporter.py:699]: (I admit that I do not remember exact behaviour of `message` field, would need to look at other code or test it)

Comment (KasperFranz) [reply wikibrain/wikimedia_link_issue_reporter.py:699]: Ahh that makes sense - I can update that

Comment (KasperFranz) [reply wikibrain/wikimedia_link_issue_reporter.py]: That is now added

**Comment by matkoniecz** [comment]
Thanks! And in a bit unlikely case of looking for more coding tasks: test suite has some failing tests where bug was found but is not yet fixed. Some may be even fixable in reasonable way.

**Comment by matkoniecz** [comment]
Hmmmm, I see

```
======================================================================
FAIL: test_that_wikidata_works_with_multiple_brands_one_invalid (test_wikimedia_link_issue_reporter.Tests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mateusz/Documents/install_moje/OSM_software/wikibrain_py_package_published/test_wikimedia_link_issue_reporter.py", line 490, in test_that_wikidata_works_with_multiple_brands_one_invalid
    self.assertIsNone(should_be_fine)
AssertionError: <wikibrain.wikimedia_link_issue_reporter.ErrorReport object at 0x7f24aa898d90> is not None
```

Given that https://www.wikidata.org/wiki/Q7501155 is actually defunct, it should not be None, right? And if one would be invalid then error should be still emitted?

**Comment by KasperFranz** [comment]
Let me review it again - thank you for flagging

**Pull request merged**